ops/ccs-ops-misc/synthetic-data/scripts/outpatient/convert_files.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
```

Comment:
For documentation purposes, I decided to put these files into a new script directory.